### ISSUE TITLE
Support for SBT v0.13

### ISSUE THREAD

**Issue by hawkw** [philcali/sbt-lwjgl-plugin#37]
It says in the README file that SBT 0.13 is not currently supported. Is there a specific reason why this is the case? 

I'd like to be able to use this plugin with a project that I believe also currently requires some v0.13-specific features. I'd be happy to fork the repo and take a crack at making it v0.13-compatible, but I'm not sure what exactly the issue is.

**Comment by philcali** [comment]
I'm assuming that note in the README exists solely because it remains unpublished. I just built it locally with no problem.

If the `addSbtPlugin` command is failing to resolve, then I would suggest using the plugin runtime dependency path in `probject/project/Plugin.scala` until it has been correctly published:

``` scala
import sbt._
import Keys._

object Plugins extends Build {
  lazy val root = project.in(file(".")).dependsOn(lwjglPlugin)
  lazy val lwjglPlugin = uri("git://github.com/philcali/sbt-lwjgl-plugin.git")
}
```
 
I think only a few things have changed since 3.1.4, so I guess it's worthy of a minor version bump.

**Comment by hawkw** [comment]
Ah, excellent, thank you! I'll see if it works for me.

**Comment by hawkw** [comment]
Yup, it looks like `addSbtPlugin` doesn't resolve correctly, so I added it in `project` as you recommended. Everything appears to work correctly so far.

**Comment by nstoddard** [comment]
Is there any update on this? I'm having difficulties setting it up like @philcali suggested since I'm not very familiar with SBT, but I don't want to be stuck with Java 7 (older versions of SBT don't support Java 8).

**Comment by gaborigloi** [comment]
Hi @nstoddard , the following configuration worked for me, with sbt 0.13.11 (I added the plugin repository as described in the [sbt documentation](http://www.scala-sbt.org/0.13/docs/Plugins.html#Using+a+library+in+a+build+definition+example)):

project/plugins.sbt

``` sbt
lazy val root = (project in file(".")).dependsOn(lwjglPlugin)
lazy val lwjglPlugin = uri("git://github.com/philcali/sbt-lwjgl-plugin")
```

build.sbt

``` sbt
seq(lwjglSettings: _*)
```

Hope this helps.
